packages/core-manager/src/defaults.ts

```diff
@@ -16,6 +16,17 @@ export const defaults = {
             },
         },
     },
+    connection: {
```

Comment:
This should be automatically handled. There is no point in the user defining duplicated data because the same database as the relay process has to be used to get the data.